src/server/model_run.R

```diff
@@ -30,12 +30,27 @@ modelRun <- function(input, output, spectrumFilesState, surveyAndProgramData) {
             state$state <- "finished"
 
         }
-
     })
 
     output$modelRunState <- shiny::reactive({ state$state })
     shiny::outputOptions(output, "modelRunState", suspendWhenHidden = FALSE)
 
+    shiny::observeEvent(surveyAndProgramData$survey, {
+        if (surveyAndProgramData$surveyTableChanged > 1){
```

Comment:
We only reset the state if this isn't the first change to `survey` - this is because a change event will occur the first time the user navigates to the input data page, but this first change event doesn't represent a change to the data. Similarly for `program_wide` below.

Reply:
Maybe add this to the code as a comment?